client/src/components/screens/PatientVariant/components/VariantNavigation.jsx

```diff
@@ -154,6 +161,29 @@ class VariantNavigation extends React.Component {
     ));
   }
 
+  handleMenuSelection(e) {
+    let { activeMenu } = this.state;
+    activeMenu = activeMenu[0] === e.key ? [] : [e.key];
```

Comment:
It is preferable to not directly modify members of the state. They should be immutable. Otherwise in some cases this can lead to some very nasty bugs that are hard to track. Better create a copy then call setState after. For instance

```
     const ( activeMenu } = this.state;
     let activeMenuClone = deepClone(activeMenu);
     activeMenuClone = activeMenuClone[0] === e.key ? [] : [e.key];
```

Reply:
I'll leave it as it is more stylistic, but technically a deep clone was not required here as the object itself is never modified does not change (the only thing that changes is the value of a local variable)

This would have been the purist approach here (if changing variable was the worry):

```
const { activeMenu } = this.state;
const = newActiveMenu === e.key ? [] : [e.key]
this.setState({
    activeMenu: newActiveMenu,
});
```

Reply:
You're right. The 'let' instead of 'const' had me misread the code. 
the 'deepClone' is not necessary. Only changing 'let' to 'const' in this case as good practice.
@claudia1296 @Magnitus-